config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/util/paths/PathUtils.java

```diff
@@ -40,4 +40,14 @@ public static String unixPathEndingWithSeparator(@NotNull String first, String..
     public static String unixPathEndingWithSeparator(@NotNull Path path) {
         return separatorsToUnix(path.toString()) + UNIX_PATH_SEPARATOR;
     }
+
+    public static String extractName(String nameWithPath) {
+        int index = nameWithPath.lastIndexOf(UNIX_PATH_SEPARATOR);
+        return index > -1 ? nameWithPath.substring(index + 1) : nameWithPath;
```

Comment:
Interesting. I've never used condition like above. I always go with the != w.r.y. -1. 
Never mind.